migrations/20240103051259_welcome_notification.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE welcome_notification (
+    id              uuid         PRIMARY KEY DEFAULT gen_random_uuid(),
+    created_at      timestamptz  NOT NULL DEFAULT now(),
+    updated_at      timestamptz  NOT NULL DEFAULT now(),
+    project         uuid         NOT NULL REFERENCES project (id) ON DELETE CASCADE,
+    type            uuid         NOT NULL,
+    title           varchar(255) NOT NULL,
+    body            varchar(255) NOT NULL,
+    url             varchar(255) NULL,
+
+    UNIQUE (project)
+);
+CREATE INDEX welcome_notification_project_idx ON welcome_notification (project);
```

Comment:
There is `UNIQUE` index that's already defined for the `project`, so we don't need to add an additional index to speedup lookups for the `project`.

Reply:
Good point, fixed in 8eb7dadc522f7e8e7b14314d371efa9378b6cde5